codespell_lib/_codespell.py

```diff
@@ -416,7 +416,7 @@ def parse_options(args):
 def build_exclude_hashes(filename, exclude_lines):
     with codecs.open(filename, 'r') as f:
         for line in f:
-            exclude_lines.add(line)
+            exclude_lines.add(line.rstrip())
```

Comment:
This will remove any whitespace, not just the line ending. I'm not sure if that's necessarily an issue, but it's a bigger change than the minimum needed to fix this, so we should possibly have some whitespace test cases too.